build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1126,6 +1156,19 @@ message GetActionResultRequest {
   // The digest of the [Action][build.bazel.remote.execution.v2.Action]
   // whose result is requested.
   Digest action_digest = 2;
+
+  // A hint to the server to inline stdout in the
+  // [ActionResult][build.bazel.remote.execution.v2.ActionResult] message.
```

Comment:
inline -> request inlining ? I think that helps reinforce the message that the server may not comply.

Reply:
done